static/app/views/settings/projectDebugFiles/externalSources/customRepositories/details.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from '@emotion/styled';
+
+// import DateTime from 'app/components/dateTime';
```

Comment:
```suggestion
```

Reply:
I will enable this as soon as the backend provides the correct data format